### ISSUE TITLE
Stabilize integration tests by removing use of stale element?

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-gui#5514]
As discussed in https://github.com/LLK/scratch-gui/issues/5429#issuecomment-598543870, the change in https://github.com/LLK/scratch-gui/pull/4871 introduced an experiment: each time we found an element by XPath, we would then try repeatedly calling `isDisplayed()` on the element until it became visible.

This usually works fine, but sometimes -- such as when first finding sections of the blocks menu/palette, which rapidly replaces itself twice on initialization -- the element has been removed by the time we're calling `isDisplayed()` on it.

We should try removing this `isDisplayed()` loop, and seeing if the tests seem more stable or less stable afterwards. Since this was always an experiment, and we've identified at least one case where it specifically keeps a test from working, I think we should err on the side of removing it.

**Comment by adroitwhiz** [comment]
If calling `isDisplayed` on the element causes it to error out, wouldn't all other attempts to use the element fail as well since it's stale?

**Comment by adroitwhiz** [comment]
Managed to get a `StaleElementError` even after reverting #4871's changes--looks like another workaround is needed

**Comment by benjiwheeler** [comment]
> If calling isDisplayed on the element causes it to error out, wouldn't all other attempts to use the element fail as well since it's stale?

Yes, true. We need some way to keep retrying until the element's presence is stable across more than a small amount of time. Not sure if that is possible.

**Comment by adroitwhiz** [comment]
I think a well-placed `sleep` call would work around the issue--the trouble is getting it to run *after* the loading screen goes away-- [as I mentioned here](https://github.com/LLK/scratch-gui/pull/5515#issuecomment-598833694):
> I tried adding a driver.sleep(500) above the problematic clickText, and it didn't help either.
> It looks like there's currently no system for telling when the GUI has loaded, and hence no way to sleep for a certain amount of time before clicking the first element. I assume that's why my sleep call failed-- it was sleeping for 500ms at the beginning of the page load, then going on to wait for the block palette to appear before the loader had even gone away.

That's essentially what I'm trying to do in https://github.com/LLK/scratch-gui/pull/5516.

**Comment by benjiwheeler** [comment]
> Managed to get a StaleElementError even after reverting #4871's changes--looks like another workaround is needed

Agreed.
